go.mod

```diff
@@ -73,6 +75,11 @@ require (
 	golang.org/x/tools v0.22.0 // indirect
 	golang.org/x/xerrors v0.0.0-20231012003039-104605ab7028 // indirect
 	google.golang.org/genproto/googleapis/rpc v0.0.0-20240227224415-6ceb2ff114de // indirect
+	gopkg.in/inf.v0 v0.9.1 // indirect
 	gopkg.in/yaml.v2 v2.4.0 // indirect
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 )
+
+require github.com/aws/aws-sdk-go v1.34.28
+
+require github.com/gocql/gocql v1.6.0
```

Comment:
Can we use latest versions? https://github.com/scylladb/gocql/releases

Reply:
Run `go mod tidy` after.

Reply:
Done